src/Symfony/Component/Notifier/Bridge/Amazon/composer.json

```
@@ -0,0 +1,32 @@
+{
+    "name": "symfony/amazon-notifier",
+    "type": "symfony-bridge",
+    "description": "Provides Amazon SNS integration for Symfony Notifier.",
+    "keywords": ["sms", "amazon", "sns", "notifier"],
```

```suggestion
    "keywords": ["amazon", "sns", "notifier"],
```

Its about chat, not sms, isn't it?

---

Its about sms and chat